### PR TITLE
feat: Release settings page

### PR DESCRIPTION
**Pull request by simotae14**
### What does it do?

Add the Release Settings page to select the timezone to use
Schermata 2024-06-17 alle 10 22 06

### Why is it needed?

We want to give the ability to the user to select the default timezone to use in Releases

Page should consume these two endpoints:

Get settings
GET `/releases/settings`

Update settings
PUT `/releases/settings`

body:
```
{
  defaultTimezone: string | null
}
```

### How to test it?

Open the Strapi CMS with a EE license
- Go to the "Settings" and open the Release link under the "Global Settings"
- Select a Timezone from the dropdown and save it
  - then after the Successful message you can check if the value is saved just refreshing the page

### Related issue(s)/PR(s)

CS-811

### CODE REVIEW

File: packages/core/content-releases/admin/src/pages/SettingsPage.tsx [hunk 1]
```diff
@@ -15,31 +82,112 @@ export const SettingsPage = () => {
...
+            return (
+              <>
+                <Layouts.Header
+                  primaryAction={
+                    <Button
+                      disabled={!modified || isSubmittingForm}
+                      loading={isSubmitting}
+                      startIcon={<Check />}
+                      type="submit"
+                      size="L"
```
Comment (remidej) [inline packages/core/content-releases/admin/src/pages/SettingsPage.tsx:108]: I don't think size L is needed, the button doesn't look this big to me in the designs

File: packages/core/content-releases/admin/src/components/ReleaseModal.tsx [hunk 2]
```diff
@@ -248,7 +248,7 @@ interface ITimezoneOption {
   value: string;
 }
 
-const getTimezones = (selectedDate: Date) => {
+export const getTimezones = (selectedDate: Date) => {
```
Comment (remidej) [inline packages/core/content-releases/admin/src/components/ReleaseModal.tsx:251]: Maybe this belongs in the utils folder?

**Review by remidej — COMMENTED** [review]
Tested ✅

File: packages/core/content-releases/admin/src/pages/SettingsPage.tsx (hunk 1 above)
Comment (simotae14) [reply packages/core/content-releases/admin/src/pages/SettingsPage.tsx:108]: ok

File: packages/core/content-releases/admin/src/components/ReleaseModal.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ import { useLocation } from 'react-router-dom';
 
 import { RELEASE_SCHEMA } from '../../../shared/validation-schemas';
 import { pluginId } from '../pluginId';
-import { getTimezoneOffset } from '../utils/time';
+import { getTimezones } from '../utils/getTimezones';
```
Comment (remidej) [inline packages/core/content-releases/admin/src/components/ReleaseModal.tsx:26]: sorry to be annoying but I think getTimezones should be inside that utils/time, they're closely related

**Review by remidej — APPROVED** [review]
Tested again ⭐

**Pull request merged**